### ISSUE TITLE
java.lang.NoClassDefFoundError: org/apache/tools/ant/util/FileUtils

### ISSUE THREAD

**Issue by bruderstein** [jballant/CommonJSAutoComplete#3]
On WebStorm 9.0.3, I get a NoClassDefFoundError for FileUtils when I get to the "r" of require (`var myModule = r`)

I presume I'm missing the ant-util JAR somewhere, but I don't know where I need to put it or do to configure it's path.

```
org/apache/tools/ant/util/FileUtils
java.lang.NoClassDefFoundError: org/apache/tools/ant/util/FileUtils
    at completion.util.StringUtil.relativizePaths(StringUtil.java:69)
    at completion.JSRequirePathFinder.getRequirePathRelativeToCurrentFile(JSRequirePathFinder.java:112)
    at completion.JSRequirePathFinder.findPathsForVarName(JSRequirePathFinder.java:85)
    at completion.JSRequireCompletionProvider.addCompletions(JSRequireCompletionProvider.java:77)
    at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:36)
    at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:154)
    at completion.JSRequireCompletionContributer.fillCompletionVariants(JSRequireCompletionContributer.java:34)
    at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:84)
    at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:135)
    at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:128)
    at com.intellij.codeInsight.template.impl.LiveTemplateCompletionContributor$1.addCompletions(LiveTemplateCompletionContributor.java:75)
    at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:36)
    at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:154)
    at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:84)
    at com.intellij.codeInsight.completion.CompletionService.performCompletion(CompletionService.java:115)
    at com.intellij.codeInsight.completion.CompletionProgressIndicator.a(CompletionProgressIndicator.java:788)
    at com.intellij.codeInsight.completion.CompletionProgressIndicator.access$1000(CompletionProgressIndicator.java:89)
    at com.intellij.codeInsight.completion.CompletionProgressIndicator$1CalculateItems.run(CompletionProgressIndicator.java:770)
    at com.intellij.codeInsight.completion.AsyncCompletion$1$1$1.run(CompletionThreading.java:93)
    at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:908)
    at com.intellij.codeInsight.completion.AsyncCompletion$1$1.run(CompletionThreading.java:88)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl$3.run(ProgressManagerImpl.java:194)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.a(ProgressManagerImpl.java:281)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:233)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcess(ProgressManagerImpl.java:181)
    at com.intellij.codeInsight.completion.AsyncCompletion$1.run(CompletionThreading.java:84)
    at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:406)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
    at java.util.concurrent.FutureTask.run(FutureTask.java:262)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
    at java.lang.Thread.run(Thread.java:745)
    at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:56)
Caused by: java.lang.ClassNotFoundException: org.apache.tools.ant.util.FileUtils PluginClassLoader[jballant.CommonJSAutoComplete, 0.1.3]
    at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:68)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
    ... 33 more
```

**Comment by jballant** [comment]
Thanks for letting me know. I'll get WebStorm 9.0.3 and check it out.

**Comment by jballant** [comment]
I was able to reproduce the issue and fix it on my end. 

It seems like the newest version of WebStorm doesn't provide that library automatically anymore. So I've bundled it with the plugin. 

If you'd like to try it out, here is a link to the 'master' branch zip file: https://github.com/jballant/CommonJSAutoComplete/raw/master/CommonJSAutoComplete.zip

You can install it by going to "Preferences > Plugins > Install Plugin from Disk..." and selecting that zip file.

Let me know if you discover that it still doesn't work. Otherwise, I'll release the update for the plugin when I can.

**Comment by bruderstein** [comment]
Yes, that works!  Thanks for the super quick fix.  I'll try and check it in 10 tonight at home.

**Comment by jballant** [comment]
Good to hear. The fix has been released in versions 0.1.4 and 0.1.5 which are available now.

**Comment by bruderstein** [comment]
Works in 10 too!
